tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -88,6 +88,31 @@ export class Ide {
         await this.waitNotificationAndClickOnButton(notificationText, 'Open Link', timeout);
     }
 
+    async waitDialogShellAndPerformAction(timeout: number, openLink: boolean, closeDialog: boolean) {
+        const dialogShellLocator: By = By.id('theia-dialog-shell');
```

Comment:
Looks like duplication of the "DialogWindow". I mean not all method just inner steps.
Please get rid of the duplication inside this method (reuse methods from "DialogWindow") and move 
this method to the "DialogWindow".
![Screenshot from 2020-01-22 11-56-43](https://user-images.githubusercontent.com/22724619/72884441-cb6d5d00-3d0e-11ea-9bfa-3647c622e413.png)

Reply:
I've ended up removing the method altogether as it's no longer required since I'm using the DialogWindow class now.